daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/SpeedyTest.scala

```diff
@@ -222,6 +222,14 @@ class SpeedyTest extends WordSpec with Matchers {
     }
   }
 
+  "type abstraction" should {
+    "evaluate when using (un uninhabited) type of kind '* -> nat'" in {
```

Comment:
It's the kind `* -> nat` that is uninhabited, i.e., there are not types of this kind.
```suggestion
    "evaluate when using a type variable of the uninhabited kind '* -> nat'" in {
```